client/my-sites/site-settings/settings-controller.js

```diff
@@ -54,7 +54,11 @@ export const redirectToolsIfRemoveDuplicateViewsExperimentEnabled = async ( cont
 		if ( ! URL_MAP[ slug ] ) {
 			return next();
 		}
-		return page.redirect( `/sites/settings/site/${ context.params.site_id }/${ URL_MAP[ slug ] }` );
+
+		const queryParams = context.querystring ? `?${ context.querystring }` : '';
+		return page.redirect(
+			`/sites/settings/site/${ context.params.site_id }/${ URL_MAP[ slug ] }${ queryParams }`
```

Comment:
The change looks good!

Can we test this on Calypso Live? I tried updating the `Authorize transfer` link in the email to a Calypso Live URL, but I encountered an error.

Reply:
NVM if you’ve already figured out how to make it work :)

Reply:
Sorry I didn't even think to write that out, you need to [catch the redirect](https://chromewebstore.google.com/detail/link-redirect-trace/nnpljppamoaalgkieeciijbcccohlpoh) and then go to the calypso live link manually. e.g.

https://container-pedantic-noether.calypso.live/settings/start-site-transfer/test131134.wordpress.com?site-transfer-confirm=the-confirm-code

If this redirects to /sites/settings/site/:site/transfer-site?site-transfer-confirm=the-confirm-code then it's working.